index.bs

```diff
@@ -187,6 +186,108 @@ If the console is not open when the printer operation is called,
 implementations should buffer messages to show them in the future up to an
 implementation-chosen limit (typically on the order of at least 100).
 
+<h4 id="loglevel-severity">Indicating <var>logLevel</var> Severity</h4>
+
+This section groups together {{console}} methods whose <var>logLevel</var>s produce output
+indicating similar levels of severity.
```

Comment:
This paragraph confusing because _logLevel_ is now unique per-method, whereas the "Log Level Severity" column is a grouping. Similarly the paragraph afterward doesn't make much sense to me.

Here would be my take:

> Each {{console}} method uses a unique value for the _logLevel_ parameter when calling Printer, allowing implementations to customize each printed message depending on the method from which it originated. However, it is common practice to group together certain methods and treat their output similarly, in four broad categories. This table summarizes these common groupings:

(the table goes here. Maybe the "Log Level Severity" column gets renamed "Grouping" and goes first.)

> These groupings are meant to document common practices, and do not constrain implementations from providing special behavior for each method, as in the following examples: